plugins/obh-providers/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "obh-providers",
+  "version": "1.0.0",
+  "description": "Sources a list of OBH providers from CommCareHQ",
+  "main": "index.js",
+  "scripts": {}
+}
```

Comment:
Interesting, you need to add it both as a gatsby plugin and as an npm package?

Reply:
yeah that's right: https://www.gatsbyjs.com/docs/tutorial/part-3/#add-a-plugin-to-your-site